### PR TITLE
Keb context updates

### PR DESCRIPTION
**Pull request by piotrmiskiewicz**
**Description**

Changes proposed in this pull request:
- service contains a flag "allow_context_updates" which enables context updates by KEB client
- added more logging to the update endpoint logic to see what we get

### CODE REVIEW

File: components/kyma-environment-broker/internal/broker/instance_update.go [hunk 1]
```diff
@@ -2,26 +2,60 @@ package broker
...
+	var ersContext internal.ERSContext
+	err = json.Unmarshal(details.RawContext, &ersContext)
+	if err != nil {
+		logger.Errorf("unable to decode context: %s", err.Error())
+		return domain.UpdateServiceSpec{}, nil
+	}
+	logger.Infof("Global account ID: %s active: %v", instance.GlobalAccountID, ersContext.Active)
+
+	var contextData map[string]interface{}
+	_ = json.Unmarshal(details.RawContext, contextData)
```
Comment (kjaksik) [inline components/kyma-environment-broker/internal/broker/instance_update.go:50]: Why don't we handle this error?

Comment (kjaksik) [reply components/kyma-environment-broker/internal/broker/instance_update.go:50]: Btw my IDE says that `contextData` should be a pointer 😄

File: components/kyma-environment-broker/internal/broker/instance_update.go [hunk 2]
```diff
@@ -2,26 +2,60 @@ package broker
...
-	b.log.Infof("Update instanceID: %s", instanceID)
-	b.log.Infof("Update details: %+v", details)
-	b.log.Infof("Update asyncAllowed: %v", asyncAllowed)
+	logger := b.log.WithField("instanceID", instanceID)
+	logger.Infof("Update instanceID: %s", instanceID)
+	logger.Infof("Update asyncAllowed: %v", asyncAllowed)
+
+	instance, err := b.instanceStorage.GetByID(instanceID)
+	if err != nil {
+		logger.Errorf("unable to get instance")
```
Comment (kjaksik) [inline components/kyma-environment-broker/internal/broker/instance_update.go:36]: What do you think about logging the error that we get from the `GetByID` method too?

File: components/kyma-environment-broker/internal/broker/instance_update.go (hunk 1 above)
Comment (piotrmiskiewicz) [reply components/kyma-environment-broker/internal/broker/instance_update.go:50]: the second unmarshalling is just for a POC, but will handle the error

**Review by kjaksik — APPROVED** [review]

**Pull request merged**